view/frontend/web/js/model/fastlane.js

```diff
@@ -1,7 +1,6 @@
 define([
     'ko',
-    'Bold_CheckoutPaymentBooster/js/action/general/load-script-action',
-    'prototype'
```

Comment:
Please ensure that we are not using Prototype or `$` elsewhere in this logic.

Reply:
I've tested the express pays and checkout with paypal, google, and CC without issue.

Reply:
Actually fastlane doesn't work properly without prototype right now.

Reply:
Replaced the prototype wrap function with overriding the require load function to add the sdk client token and metadata id for fastlane script. 
Tested with the express + wallet pays with and without fastlane enabled.